scripts/compose.py

```diff
@@ -886,6 +886,69 @@ def _content(self):
         )
         return content
 
+class OpbeansJava(Service):
+    SERVICE_PORT = 3002
+    DEFAULT_AGENT_BRANCH = "master"
+    DEFAULT_AGENT_REPO = "elastic/apm-agent-java"
+    DEFAULT_LOCAL_REPO = "."
+
+    @classmethod
+    def add_arguments(cls, parser):
+        super().add_arguments(parser)
```

Comment:
we've decided to make things mostly work for python2 - can you change this to `super(OpbeansJava, self).__init__(**options)`

Reply:
ha, sorry I didn't get the notification.  Fixing it.